R/scrape_usafacts.R

```diff
@@ -0,0 +1,46 @@
+usa_facts_data <- function(...) UseMethod("usa_facts_data")
```

Comment:
Hi Connor, the script that you wrote was awesome! I cleaned it up a little and integrated into the app.R, so that when we run the app.R code it should now be dumping the new file directly into the data/usa_facts data folder.

However, this isn't even necessary, because I made the default function return the file and that's what gets pulled into app.R

I re-wrote the function in an "object-orientedesque" way. I think when the do convert it to an api this will be very easy for us to update. I'm not sure if you've seen or written R code in this way before. I've found it can be a really useful way to organize my processes. Basically I've been making a default function which does the heavy lifting. Then the method functions (usa_facts_data.deaths and usa_facts_data.confirmed) essentially just add different parameters to the main function.

I re-ran all the code and it looks to me like everything is working correctly!

Reply:
Nice! I spent some time looking at the code structure and it makes sense to me. I was actually also working on restructuring the scrape_usafacts script so that all of the functions were nested. It was similar to yours but not object-oriented. Thanks!